### PR TITLE
[SHAANTHINI T]-[USER-ACCOUNT-ADDED_OTP-POPUP-UI]

### PR DESCRIPTION
**Pull request by ShaanthiniThiyagarajan**
Ticket: PROJ-XXXX <br>
Linked PRs: (links to corresponding PRs, optional)

- [ ] Browser testing (Chrome, Firefox, Safari, Edge)<br>
- [ ] Accessibility testing (NVDA + Firefox, Voiceover + Safari)<br>
- [x] I have read and agreed to the process outlined in [Creating a Pull Request](File_Path for creatingPullRequest.md)<br> 
- [x] I ran `npm run extract-intl` to update the messages generated
- [ ] If this is a *downstream* I have double checked that my *destination branch* is my *epic branch*

## Changes

* OTP modal added

* (Add a screenshot of the code coverage of the code coverage of the new files, you changed)

## Screenshots / GIFs

* (Screenshots or GIF to show functionality)
image

### CODE REVIEW

File: app/components/Form/Validators/index.js [hunk 1]
```diff
@@ -19,3 +19,4 @@ export const lengthCheckValidator = (message, min = 0, max = 25) => (value) => v
 export const emailValidator = (message) => (value) => value && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)
   ? message
   : '';
+export const otpLengthValidator = (message) => (value) => value && value.length === 6 ? '' : message;
```
Comment (Azam4204) [inline app/components/Form/Validators/index.js]: As I could see that existing validation function `lengthCheckValidator` used to validate length of OTP input. This function now seems unused. 
Can you please confirm?

**Review by Azam4204 — COMMENTED** [review]
Asked small clarification

File: app/components/Form/Validators/index.js (hunk 1 above)
Comment (ShaanthiniThiyagarajan) [reply app/components/Form/Validators/index.js]: Will remove it

**Review by Azam4204 — APPROVED** [review]
LGTM

**Review by Azam4204 — APPROVED** [review]
Need minor simplifying changes. Will address in different work PR. Merging this for now

**Pull request merged**